images/capi/ansible/roles/kubernetes/tasks/url.yml

```diff
@@ -20,9 +20,29 @@
     force: yes
   loop:
   - ctr
-  - crictl
```

Comment:
The Download and Install part can be above this so that crictl gets copied to /usr/bin as well

Reply:
Ack